### PR TITLE
added button to clear server cache

### PR DESCRIPTION
**Pull request by ssuvorov-fls**
https://github.com/OHDSI/WebAPI/issues/1322

### CODE REVIEW

File: js/pages/configuration/configuration.js [hunk 1]
```diff
@@ -130,6 +132,20 @@ define([
...
+      cacheApi.clearCache().then(() => {
+        alert("Server cache has been cleared.");
+      });
+    };
+    
+    canClearServerCache() {
+      if (!config.userAuthenticationEnabled) {
+        return false;
+      } else {
+        return (config.userAuthenticationEnabled && this.isAuthenticated() && authApi.isPermittedClearServerCache());
```
Comment (wivern) [inline js/pages/configuration/configuration.js]: Does the simple `config.userAuthenticationEnabled && (this.isAuthenticated() && authApi.isPermittedClearServerCache())` not enough?

File: js/pages/configuration/configuration.js [hunk 2]
```diff
@@ -130,6 +132,20 @@ define([
 			alert("Local Storage has been cleared.  Please refresh the page to reload configuration information.")
 		};
 
+		clearServerCache() {
+      cacheApi.clearCache().then(() => {
```
Comment (wivern) [inline js/pages/configuration/configuration.js]: Let\s wrap this with confirmation

File: js/pages/configuration/configuration.js (hunk 1 above)
Comment (ssuvorov-fls) [reply js/pages/configuration/configuration.js]: agree, fixed

File: js/pages/configuration/configuration.js (hunk 2 above)
Comment (ssuvorov-fls) [reply js/pages/configuration/configuration.js]: done

**Pull request merged**